L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -646,10 +650,16 @@ namespace trklet {
 
     unsigned int writememsect_{3};  //writemem only for this sector (note that the files will have _4 extension)
 
-    bool writeTripletTables_{false};  //Train and write the TED and TRE tables. N.B.: the tables
-                                      //cannot be applied while they are being trained, i.e.,
-                                      //this flag effectively turns off the cuts in
-                                      //TrackletEngineDisplaced and TripletEngine
+    //FIXME: The TED and TRE tables are currently under development. They use
```

Comment:
What does FIXME mean?  This PR is going to be followed by another PR on this code?
If not, and you are suggesting that TED and TRE tables might/will undergo changes (content/format) please remove this comment as it introduces confusion.

Reply:
There likely will be changes to the code as we continue to develop the tables, but not in the immediate future. So I removed the comment.